### PR TITLE
New View Claims Page

### PR DESCRIPTION
**Pull request by sloan-seamus**
Revamped the View Claims page.
Claims are now being sorted by their status.

Work in progress. Unsure of how to sort upon load up.

### CODE REVIEW

File: frontend/src/app/view-claims/view-claims.component.ts [hunk 1]
```diff
@@ -27,6 +25,10 @@ export class ViewClaimsComponent implements OnInit {
   readonly classes = this._theme.addStyleSheet(STYLES);
 
   private claims: Protos.Claim[] = [];
+  private claims0: Protos.Claim[] = [];
+  private claims1: Protos.Claim[] = [];
```
Comment (dsifford) [inline frontend/src/app/view-claims/view-claims.component.ts]: no

File: frontend/src/app/view-claims/view-claims.component.ts [hunk 2]
```diff
@@ -56,8 +54,22 @@ export class ViewClaimsComponent implements OnInit {
       () => {});
   }
 
-  openSnackBar() {
-
-
+  sortClaims(num): void{
```
Comment (dsifford) [inline frontend/src/app/view-claims/view-claims.component.ts]: Delete this entire method and instead just filter by status on click.

So, if they clicked status 2....

```ts
this.visibileClaims = this.claims.filter(claim => claim.status === 2)
```

**Review by dsifford — CHANGES_REQUESTED** [review]

File: frontend/src/app/view-claims/view-claims.component.ts (hunk 2 above)
Comment (sloan-seamus) [reply frontend/src/app/view-claims/view-claims.component.ts]: I left the method there, but instead it returns a sorted array of the claims. The for loop in the .html goes through the sorted list.

File: frontend/src/app/view-claims/view-claims.component.ts (hunk 1 above)
Comment (sloan-seamus) [reply frontend/src/app/view-claims/view-claims.component.ts]: Agreed...

File: frontend/src/app/view-claims/view-claims.component.html [hunk 3]
```diff
@@ -1,15 +1,68 @@
...
+            <p [lyTyp]="'body1'">Vehicle Color: {{ claim.vehicle.color }}</p>
+            <p [lyTyp]="'body1'">Claim Status: {{ claim.status }}</p>
+          </ly-paper>
+        </ly-grid>
+      </ly-grid>
+  </ly-tab>
+</ly-tabs>
+
+<div>
   <ng-template #noClaims>
```
Comment (sloan-seamus) [inline frontend/src/app/view-claims/view-claims.component.html:66]: Possible change here would be to say "No claims are currently awaiting settlement options" when there are claims, but no claims in that status.

**Review by dsifford — APPROVED** [review]

**Pull request merged**